language/stdlib/modules/LibraAccount.move

```diff
@@ -1051,126 +1060,105 @@ module LibraAccount {
         let transaction_sender = Signer::address_of(sender);
 
         // Check that the chain ID stored on-chain matches the chain ID specified by the transaction
-        assert(ChainId::get() == chain_id, PROLOGUE_EBAD_CHAIN_ID);
+        assert(ChainId::get() == chain_id, Errors::invalid_argument(PROLOGUE_EBAD_CHAIN_ID));
 
         // Verify that the transaction sender's account exists
-        assert(exists_at(transaction_sender), PROLOGUE_EACCOUNT_DNE);
+        assert(exists_at(transaction_sender), Errors::invalid_argument(PROLOGUE_EACCOUNT_DNE));
 
         // We check whether this account is frozen, if it is no transaction can be sent from it.
         assert(
             !AccountFreezing::account_is_frozen(transaction_sender),
-            PROLOGUE_EACCOUNT_FROZEN
+            Errors::invalid_state(PROLOGUE_EACCOUNT_FROZEN)
```

Comment:
Why froizen account is of `invalid_state`?

Reply:
What else would it be? /shrug

The only one that seems close is `REQUIRES_ROLE`, but that doesn't seem right. State seems like the best fit given the description in `Errors.move`

Maybe @tzakian or @wrwg could chime in

Reply:
I think `INVALID_STATE` makes sense here (the account is in an invalid state to send a transaction).